project/Dependencies.scala

```diff
@@ -78,7 +78,8 @@ object Dependencies {
 
   val resolutionRepos = Seq(
     // Redshift native driver
-    ("redshift" at "http://redshift-maven-repository.s3-website-us-east-1.amazonaws.com/release").withAllowInsecureProtocol(true)
+    ("redshift" at "http://redshift-maven-repository.s3-website-us-east-1.amazonaws.com/release").withAllowInsecureProtocol(true),
+    ("Snowplow Analytics Maven repo" at "http://maven.snplow.com/releases/").withAllowInsecureProtocol(true)
```

Comment:
Why was this line needed?  I thought everything was in sonatype.

Reply:
Without this line I get:

```sbt.librarymanagement.ResolveException: Error downloading com.snowplowanalytics:collector-payload-1:0.0.0```

which is needed by event generator?